src/Legacy/Drupal7.php

```diff
@@ -11,7 +11,213 @@
  * Defines the Drupal 7 legacy service.
  */
 class Drupal7 {
+
+
+  /**
+   * @method void flood_register_event
+   * @method void flood_clear_event
+   * @method bool flood_is_allowed
```

Comment:
Per http://www.phpdoc.org/docs/latest/references/phpdoc/tags/method.html

the @method calls need to be on the class.

They also need to have type hinted parameters defined.